apps/opencs/model/prefs/state.cpp

```diff
@@ -170,15 +170,15 @@ void CSMPrefs::State::declare()
         "list go to the first/last item");
 
     declareCategory ("3D Scene Input");
-    
+
     declareDouble ("navi-wheel-factor", "Camera Zoom Sensitivity", 8).setRange(-100.0, 100.0);
     declareDouble ("s-navi-sensitivity", "Secondary Camera Movement Sensitivity", 50.0).setRange(-1000.0, 1000.0);
     declareSeparator();
 
     declareDouble ("p-navi-free-sensitivity", "Free Camera Sensitivity", 1/650.).setPrecision(5).setRange(0.0, 1.0);
     declareBool ("p-navi-free-invert", "Invert Free Camera Mouse Input", false);
     declareDouble ("navi-free-lin-speed", "Free Camera Linear Speed", 1000.0).setRange(1.0, 10000.0);
-    declareDouble ("navi-free-rot-speed", "Free Camera Rotational Speed", 3.14 / 2).setRange(0.001, 6.28);    
+    declareDouble ("navi-free-rot-speed", "Free Camera Rotational Speed", 3.14 / 2).setRange(0.001, 6.28);
```

Comment:
Might be better to use `M_PI_2` or `M_PI / 2` constant instead of `3.14 / 2` and `2 * M_PI - 0.001` instead of `6.28`  for readability. Not sure does it important to exclude `2 * M_PI` value from range.

Reply:
Oh.. This is an automatic change done by Atom, using 3.14 or 6.28 isn't related to this edit. This change already got through in another commit, it does nothing but removes a few extra spaces, so I let it be there even if it makes this PR a bit more messy. I think adding an extra commit to revert those extra spaces back is unnecessary. Even if the review comment is actually worth thinking about, I won't touch the pi value here, as it's not related to this PR anyway.